bee-common/src/logger/config.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+use crate::logger::LOGGER_STDOUT_NAME;
+
+use log::LevelFilter;
+use serde::Deserialize;
+
+/// Default value for the color flag.
+const DEFAULT_COLOR: bool = true;
```

Comment:
that's a confusing `const` name, because one would expect an actual default color. I suggest "DEFAULT_USE_COLORS", or "DEFAULT_USE_COLORED_OUTPUT"

Reply:
`IS_DEFAULT_COLOR`?

Reply:
Or `DEFAULT_COLOR_ENABLED`